app/controllers/admin/api/personal/access_tokens_controller.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class Admin::Api::Personal::AccessTokensController < Admin::Api::Personal::BaseController
```

Comment:
I wonder what is the `access_token_scopes` of this one?
`:account_management` only right?

Reply:
`personal` scope sounds good to me.

Reply:
Right now it is `:account_management`.
https://github.com/3scale/porta/blob/58ada49045c93764003ebce3a8caf863c903b306/app/controllers/admin/api/base_controller.rb#L24

What it should be, I dunno 🤔 I guess `:personal` makes sense. Let's see what the rest of the team thinks. Specially @thomasmaas

Reply:
Thomas also brought up (or at least from what I understood) that that the name `personal` may be confusing because all access tokens are personal 😄

Reply:
if there's gonna be more and more personal related endpoints i think it's a good idea to make a new scope for that.. doesn't matter on a name..

Reply:
I like the idea of namespacing access tokens under "personal". Although, TBH I would prefer `my` rather than `personal`, so it would be `/admin/api/my/access_tokens.json` instead of `admin/api/personal/access_tokens.json`.

More than "account management", that's actually "self management" I think; and perhaps the same applies to Personal details and to Notification preferences – `/admin/api/my/personal_details.json`, `/admin/api/my/notification_preferences.json`.

Whenever a user performs actions over entities that only exist under his/her own account, he/she should not be required to know his/her own user ID to do it. The URLs should be more straightforward and user ID inferred from the access token used to perform the action.

That's one thing. Another thing is having a "Personal API" to group and separate all those endpoints.

I'm assuming that another admin user could also perform actions over the same resources belonging to his/her provider admin colleague, right? From the perspective of that other admin, the actions are not "self management" but indeed "account management", and they do require the user ID to be in the URL.

So, for some resources/actions, will we have two different endpoints, one for account management (with user ID in the URL) and another for self management (without user ID the URL), that do exactly the same thing. Correct? Having a "Provider API" (or "My API" or whatever) API group defined, for every pair of endpoints that do the same thing, one will be under "Provider API" while the other will remain under "Account Management API", right?

Can we apply different security policies for each one of those groups? Like, all access tokens should have RW access over whatever goes under "Provider API" I guess, while specific access tokens may have RW (or RO) access to Account Management API.

To sum up, if having the Provider API group can help us manage permissions over Account management completely independently from the permissions over Provider API, even though, in the end, some of endpoints perform exactly the same actions over the same resources, then YES. Let's create Provider API.

Reply:
> Are we sure it makes sense to have a special scope for personal? Why?

I can explain:

A member which does not have `account_management` access 
![image](https://user-images.githubusercontent.com/64276/51530711-dcee0180-1e3b-11e9-83f7-79017a69c2af.png)

Should still be able to create access tokens:
![image](https://user-images.githubusercontent.com/64276/51530744-f000d180-1e3b-11e9-9bf0-486a8cd5228c.png)

Reply:
Very true

Reply:
@guicassolato @hallelujah and I spoke in a call and the conclusion is:
If product agrees, let's merge this for now and we can improve it later.